### PR TITLE
ZUR-184 feat: added functionality to add reaction to chats and images in chat rooms

### PR DESCRIPTION
**Pull request by Johndiddles**
Screenshot 2022-11-12 at 04 22 15

### CODE REVIEW

File: frontend/zc_messaging/src/pages/message-board/index.jsx [hunk 1]
```diff
@@ -170,11 +173,28 @@ const MessagingBoard = () => {
         emoji: emoji,
         reactedUsersId: [currentUserId]
       }
-      message.emojis.push(newEmojiObject)
+      let emojisArray = [...message.emojis, newEmojiObject]
+      // message.emojis = emojisArray
```
Comment (JulianaSau) [inline frontend/zc_messaging/src/pages/message-board/index.jsx]: If the code isn't being used, you can do away with it entirely. Same applies to the rest of this file. If there's any code that you commented out that you know cannot be used in the future, do away with it

**Review by JulianaSau — CHANGES_REQUESTED** [review]

**Review by JulianaSau — COMMENTED** [review]
Also, add a video instead of a screenshot showing you adding the emojis to the message

**Comment by Johndiddles** [comment]
https://user-images.githubusercontent.com/48492384/201481927-07e2a5b5-88f2-4921-9fd2-398ab53c1b6e.mp4

I have removed the unused codes and attached a demo video illustrating what this PR does

**Comment by JulianaSau** [comment]
@tundealabi

**Comment by tundealabi** [comment]
https://user-images.githubusercontent.com/61058516/201493764-83977c24-c9aa-4a7c-9f9d-098ca77a26b3.mov

Yo @Johndiddles can't get this to work. Also, I think you need to run zuri messaging in the context of zuri main not as a standalone

**Comment by tundealabi** [comment]
So you need to also clone the zuri main repo, run both projects locally.
Head over to `./packages/root-config/src/assets/plugin.importmap.json` and add this `  "@zuri/zuri-plugin-messaging": "http://localhost:5013/zuri-messaging.js"`

**Comment by Johndiddles** [comment]
@tundealabi and @JulianaSau  I have made a fix for this on both zc_main and zc_messaging and here is a video of me testing it

https://user-images.githubusercontent.com/48492384/201506673-01971d17-a1aa-47ed-8fa1-4b799e83ae83.mov

**Comment by JulianaSau** [comment]
Did you follow the instructions above and run zc_messaging in the context of zc_main?

**Comment by Johndiddles** [comment]
Yes, I have now made a patch for the suggestions @tundealabi made earlier today.

**Comment by Johndiddles** [comment]
@JulianaSau @tundealabi 

https://user-images.githubusercontent.com/48492384/201535805-9a2d259a-44b5-48c8-9226-598998270760.mp4

**Comment by tundealabi** [comment]
Screenshot 2022-11-13 at 20 14 10
can add and remove emojis. @JulianaSau @Johndiddles

**Comment by tundealabi** [comment]
but I can't react to an emoji already added.. Not sure what the scope of the issue is though maybe that's for someone else to fix

**Comment by tundealabi** [comment]
when I click on an emoji added by someone else, it throws an error

**Comment by JulianaSau** [comment]
> but I can't react to an emoji already added.. Not sure what the scope of the issue is though maybe that's for someone else to fix

Yeah. That can be fixed by someone else in the future

**Review by tundealabi — APPROVED** [review]
cool!

**Review by JulianaSau — APPROVED** [review]

**Review by zxenonx — APPROVED** [review]

**Pull request merged**